docs/setup/memfootprint-test/Dockerfile

```diff
@@ -0,0 +1,3 @@
+FROM ubuntu:16.04
+COPY ./infinite.exe /.
+CMD ["./infinite.exe"]
```

Comment:
Windows `.exe`? Can we build it from source? I don't think upload a binary file along with source code is a good idea.